src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -110,51 +110,286 @@ export class MgtPersonCard extends MgtTemplatedComponent {
+    `;
+  }
+
+  /**
+   * Render the various icons for contacting the person.
+   *
+   * @protected
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected renderContactIcons(person?: IDynamicPerson): TemplateResult {
+    if (this.isExpanded) {
+      return html``;
+    }
+
+    person = person || this.personDetails;
+    const userPerson = person as MicrosoftGraph.User;
+
+    let chat: TemplateResult;
+    let email: TemplateResult;
+    let phone: TemplateResult;
+
+    if (userPerson.mailNickname) {
+      chat = html`
+        <div class="icon" @click=${this.chatUser}>
+          ${getSvg(SvgIcon.Chat, '#666666')}
+        </div>
+      `;
+    }
+    if (getEmailFromGraphEntity(person)) {
+      email = html`
+        <div class="icon" @click=${this.emailUser}>
+          ${getSvg(SvgIcon.Email, '#666666')}
+        </div>
+      `;
+    }
+    if (userPerson.businessPhones && userPerson.businessPhones.length > 0) {
+      phone = html`
+        <div class="icon" @click=${this.callUser}>
+          ${getSvg(SvgIcon.Phone, '#666666')}
+        </div>
+      `;
+    }
+
+    return html`
+      ${chat} ${email} ${phone}
+    `;
+  }
+
+  /**
+   * Render the button used to expand the additional details.
+   *
+   * @protected
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected renderAdditionalDetailsButton(): TemplateResult {
+    return html`
+      <div class="additional-details-button" @click=${(e: Event) => this.showAdditionalDetails(e)}>
+        <svg
+          class="additional-details-svg"
+          width="16"
+          height="15"
+          viewBox="0 0 16 15"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path d="M15 7L8.24324 13.7568L1.24324 6.75676" stroke="#3078CD" />
+        </svg>
+      </div>
+    `;
+  }
+
+  /**
+   * Render additional details for the person.
+   *
+   * @protected
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected renderAdditionalDetails(person?: IDynamicPerson): TemplateResult {
+    person = person || this.personDetails;
+
+    const contactDetailsTemplate = this.renderContactDetails();
+
+    // Check for additional details template
+    let additionalDetailsTemplate: TemplateResult = null;
+    if (this.hasTemplate('additional-details')) {
+      additionalDetailsTemplate = html`
+        <div class="section-divider"></div>
+        ${this.renderTemplate('additional-details', {
+          person,
+          personImage: this.getImage()
+        })}
+      `;
+    }
+
+    return html`
+      <div class="additional-details-info">
+        ${contactDetailsTemplate} ${additionalDetailsTemplate}
+      </div>
+    `;
+  }
+
+  /**
+   * Render the contact info part of the additional details.
+   *
+   * @protected
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected renderContactDetails(person?: IDynamicPerson): TemplateResult {
```

Comment:
renderContactDetails might make sense if it's not part of renderAdditionalDetails. What do you think about this:

* renderExpandedDetails
  * renderContactDetails
  * renderAdditionalDetails

Reply:
I agree. The extra method looks better